### PR TITLE
AC Automated Test - LRAC-8146 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase [hunk 1]
```diff
@@ -174,6 +174,28 @@ definition {
 
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8146 | Test Summary: Total number of individuals is shown"
+	@ignore = "true"
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase:178]: hi @AfonsoHSantos , please remove the `@ignore = "true"` annotation.

**Comment by alexcandidos** [comment]
Aside from that, the code seems okay! :)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase (hunk 1 above)
Comment (AfonsoHSantos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase]: hi @alexcandidos, i fixed that. thanks!

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @AfonsoHSantos , please take a look on this PR/branch when possible. According to the SF failure log, your code is using an invalid path locator `ACDataControlAndPrivacy#RESQUEST_LOG`.

Thanks!

**Comment by alexcandidos** [comment]
Hi @AfonsoHSantos , please take a look at this PR/branch when possible. According to the SF failure log, your code is using an invalid path locator `ACDataControlAndPrivacy#RESQUEST_LOG`.

Thanks!

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
Hi @alexcandidos, i fixed that. Thanks!

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**